tests/components/plex/test_server.py

```diff
@@ -93,7 +93,15 @@ async def test_new_ignored_users_available(hass, caplog):
     assert len(monitored_users) == 1
     assert len(ignored_users) == 2
     for ignored_user in ignored_users:
-        assert f"Ignoring Plex client owned by '{ignored_user}'" in caplog.text
+        ignored_client = [
+            x.players[0]
+            for x in mock_plex_server.sessions()
+            if x.usernames[0] in ignored_users
+        ][0]
+        assert (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5412406)